docs/server/production-deploy.md

```diff
@@ -44,15 +46,45 @@ You will want to run your own proof of concept tests and watch for key metrics t
 At a high level, you will want to track these 3 categories of metrics:
 
 - **Service metrics**: For each request made by the service handler we emit `service_requests`, `service_errors`, and `service_latency` metrics with `type`, `operation`, and `namespace` tags.
-This gives you basic visibility into service usage and allows you to look at request rates across services, namespaces and even operations.
+  This gives you basic visibility into service usage and allows you to look at request rates across services, namespaces and even operations.
 - **Persistence metrics**: The Server emits `persistence_requests`, `persistence_errors` and `persistence_latency` metrics for each persistence operation.
-These metrics include the `operation` tag such that you can get the request rates, error rates or latencies per operation.
-These are super useful in identifying issues caused by the database.
+  These metrics include the `operation` tag such that you can get the request rates, error rates or latencies per operation.
+  These are super useful in identifying issues caused by the database.
 - **Workflow stats**: The Server also emits counters on Workflows complete.
-These are  useful in getting overall stats about Workflow completions.
-Use `workflow_success`, `workflow_failed`, `workflow_timeout`, `workflow_terminate` and `workflow_cancel` counters for each type of Workflow completion.
-They are also include the `namespace` tag.
-Additional information is available in [this forum post](https://community.temporal.io/t/metrics-for-monitoring-server-performance/536/3).
+  These are useful in getting overall stats about Workflow completions.
+  Use `workflow_success`, `workflow_failed`, `workflow_timeout`, `workflow_terminate` and `workflow_cancel` counters for each type of Workflow completion.
+  They are also include the `namespace` tag.
+  Additional information is available in [this forum post](https://community.temporal.io/t/metrics-for-monitoring-server-performance/536/3).
+
+Temporal is highly scalable due to its event sourced design — we have load tested up to 200 million concurrent workflow executions.
+Every shard is low contention by design — it is very difficult to oversubscribe to a task queue in the same cluster.
```

Comment:
if this sounds weird, this directly responds to an answer we gave Instacart https://www.notion.so/temporalio/Instacart-meeting-4-May-2021-004f011d6e054afcbae52ce15af3b0d7